### PR TITLE
Speeds up the loading of posts with lots of comments

### PR DESCRIPTION
**Pull request by pfefferle**

### CODE REVIEW

**Comment by dshanske** [comment]
Looks good on my phone...want to look in front of a computer. What prompted it?

**Comment by pfefferle** [comment]
Ob pages with more than 100 comments I ran into a Webserver timeout on notiz.blog

**Comment by dshanske** [comment]
That explains it .. I've never been that popular.

**Comment by pfefferle** [comment]
Here is the example: https://notiz.blog/contact/

File: includes/class-comment-walker.php [hunk 1]
```diff
@@ -288,22 +288,14 @@ public static function comment_query( $query ) {
 			return;
 		}
 
-		if ( is_admin() ) {
+		if ( is_admin() || ! is_singular() ) {
 			return;
 		}
 
-		if ( ! is_singular() ) {
+		if ( ! empty( $query->query_vars['type__in'] ) ) {
```
Comment (dshanske) [inline includes/class-comment-walker.php:295]: Shouldn't we exclude any type specific search?  Not just in? Like type__not_in ? or even just type?

Comment (pfefferle) [reply includes/class-comment-walker.php:295]: good point!

**Pull request merged**